lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_model_base.rb

```diff
@@ -373,9 +373,15 @@ def encode_with(coder)
 
     def init_with(coder)
       @object = self.class.service_model_name_to_model(self.class.name)&.find_by(:id => coder['id'])
+      $miq_ae_logger.warn("There is no related active record object with id='#{coder[id]}' for imported #{self.class}") if @object.nil?
```

Comment:
We do not need the quotes around the `coder['id'])` value when logging and note that the log message is passing `coder[id]` instead of `coder['id']`

```ruby
$miq_ae_logger.warn("There is no related active record object with id=#{coder['id']} for imported #{self.class}") if @object.nil?
```